pkgs/development/libraries/libtickit/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  pkg-config,
+  libtermkey,
+  unibilium,
+  libtool,
+}:
+stdenv.mkDerivation rec {
+  pname = "libtickit";
+  version = "0.4.3";
+
+  src = fetchzip {
+    url = "https://www.leonerd.org.uk/code/libtickit/libtickit-${version}.tar.gz";
+    sha256 = "sha256-dKjmHojZBPP0ZQGXsno+2nFf1GCNmljWF2FklCMGCos=";
+  };
+
+  makeFlags = ["PREFIX=$(out)"];
+
+  nativeBuildInputs = [pkg-config libtermkey unibilium libtool];
+
+  meta = with lib; {
+    description = "a terminal interface construction kit";
```

Comment:
```suggestion
    description = "A terminal interface construction kit";
```

Reply:
I've forgot to mention the capitalization issue on the commit message, but it is also fixed.